src/redux.tsx

```diff
@@ -107,6 +111,12 @@ export const reducer = (state: State = INITIAL_STATE, action: Action) => {
         showSurvey: false,
         showPrompt: false,
       };
+    case 'RESET':
+      // We don't need to reset the company API key, so we'll
+      // reset to the initial state and cherry pick that
+      const updatedState: State = { ...INITIAL_STATE };
```

Comment:
Style choice up to you but you could do `const updatedState: State = { ...INITIAL_STATE, companyAuthToken: state.companyAuthToken };` and remove the next line

Reply:
yea that's better